doc/source/whatsnew/v0.21.0.txt

```diff
@@ -387,6 +387,13 @@ New Behavior
 
 Additionally, DataFrames with datetime columns that were parsed by :func:`read_sql_table` and :func:`read_sql_query` will also be localized to UTC only if the original SQL columns were timezone aware datetime columns.
 
+.. _whatsnew_0210.api.restricting_dateoffset_keywords
+
+Restricting DateOffset Keywords
```

Comment:
ok, since we made this simpler, move to just a bullet point in Other API changes section.